docker/public/wovn_index.php

```diff
@@ -0,0 +1,38 @@
+<?php
+# Enable WOVN.php library
+require_once("WOVN.php/src/wovn_interceptor.php");
+require_once("WOVN.php/src/wovn_helper.php");
+
+$parsed_url = parse_url($_SERVER["REQUEST_URI"], PHP_URL_PATH);
+
+if ($parsed_url) {
+    $paths = wovn_helper_detect_paths(dirname(__FILE__), $parsed_url);
+
+    if (strrpos($_SERVER["REQUEST_URI"], 'custom_response') != false) {
```

Comment:
WOVN's include process depends solely on `REQUEST_URI`, which we are avoiding here because it would result in loading a non-existing file.

Reply:
`custom_response` requests will never reach this file though right? due to the htaccess rule

Reply:
Based on my testing I think the htaccess rewrite rules are cumulative. All requests eventually go through `wovn.index` and get's processed, even for `custom_response`.

Reply:
Shouldn't be if you add the `[L]` flag to terminate the rewrite rule

Reply:
Actually, we don't even need the .htaccess rewrite rule for the custom responses, as the wovn_index.php script is enough to do the redirect.